TestGame/renderer_tests.dm

```diff
@@ -73,7 +73,9 @@
 	Crossed(var/atom/movable/AM)
 		src.loc = AM
 		AM << "You picked up [src]"
-		AM.overlays += image(src.icon, AM.loc, src.icon_state)	
+		var/mutable_appearance/ma = new(src.icon, src.icon_state)
+		ma.alpha = 100
+		AM.overlays += ma
```

Comment:
You left some debug code here
```suggestion
		AM.overlays += image(src.icon, AM.loc, src.icon_state)	
```

Reply:
Thought I'd keep it in since cit was the renderer test but I can remove it.